test/e2e/rte/metrics.go

```diff
@@ -173,7 +176,7 @@ var _ = ginkgo.Describe("[RTE][Monitoring] metrics", func() {
 			rteContainerName, err := e2ertepod.FindRTEContainerName(rtePod)
 			gomega.Expect(err).ToNot(gomega.HaveOccurred())
 
-			cmd := []string{"curl", "-L", fmt.Sprintf("http://127.0.0.1:%d/metrics", metricsPort)}
+			cmd := []string{"curl", "-L", fmt.Sprintf("https://%s:%d/metrics", MetricsAddress, metricsPort)}
```

Comment:
same here, this test wants to access over plain http so let's not change the URL part. Or, like above, let's change the test accordingly (but we will need to excercise the plaintext option somehow)

Reply:
Ack, Thanks!